app/views/provider/admin/api_docs/show.html.erb

```diff
@@ -1,9 +1,3 @@
-<%= stylesheet_link_tag 'active-docs/application.css' %>
-<%= javascript_tag do %>
-  window.enableApiDocsProxy = <%= Rails.configuration.three_scale.active_docs_proxy_disabled.blank? %>;
-<% end %>
-<%= javascript_include_tag 'active-docs/application.js' %>
-
```

Comment:
You probably need to remove the `css`, but do you need to remove the `js` includes? is not that causing any error in the console?

Reply:
Yes I also thought to removed only CSS, but removing JS also not showed me any difference so I thought to remove it and then see if works well without these or not

I think I need to keep this part of code, So I will add this again
```
<%= javascript_tag do %>
  window.enableApiDocsProxy = <%= Rails.configuration.three_scale.active_docs_proxy_disabled.blank? %>;
<% end %>
```

Reply:
yes, you shouldn't remove it unless you know what is it for.

Reply:
yes added again